test/e2e/clusterlogging_test.go

```diff
@@ -49,75 +47,13 @@ func TestClusterLogging(t *testing.T) {
 
 	// run subtests
 	t.Run("clusterlogging-group", func(t *testing.T) {
+
 		t.Run("Cluster with fluentd", ClusterLoggingClusterFluentd)
 		time.Sleep(time.Minute * 1) // wait for objects to be deleted/cleaned up
 		t.Run("Cluster with rsyslog", ClusterLoggingClusterRsyslog)
 	})
 }
 
-func createRequiredClusterRoleAndBinding(f *framework.Framework, ctx *framework.TestCtx) error {
```

Comment:
this didn't break e2e tests? I would think that this would cause errors in the CLO since it wouldn't have the cluster permissions to do anything with oauthclients and priorityclasses...

Reply:
The tests were failing and I couldn't understand why.  I thought these were just duplicates for resources in the manifest but I see there are some differences.  Reverting.